src/main/java/com/mkopec/ticketbookingapp/repository/TicketTypeRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.mkopec.ticketbookingapp.repository;
+
+import com.mkopec.ticketbookingapp.domain.TicketType;
+import org.springframework.data.jpa.repository.JpaRepository;
+
```

Comment:
o, a tu nie ma adnotacji. Czy to jest OK?

Reply:
Przez przypadek ominięte (nie specjalnie). Działa tylko dlatego, że użyłem spring boota, a repozytorium dziedziczy po innych interfejsach (np CrudRepository), no i konfiguracja spring boota sobie z tym poradziła